src/useCases/authenticate.ts

```diff
@@ -121,10 +124,23 @@ const authenticate = async (
 
     if (isAuthenticated && isVerified) {
       await resetUserVerificationCode(connection, emailAddress)
+      await setFailedPasswordAttempts(connection, emailAddress, 0)
```

Comment:
Is that possible to set `failed_password_attempts` to 0 in `resetUserVerificationCode` use case? It reduces number of database queries in `authenticate` function.